texastribune/donations
# Add latest queso versions and update task names

#### What's this PR do?
1. `queso-tools` and `queso-ui` move to the latest release 
2. Adds more consistent npm script command names

#### Why are we doing this? How does it help us?

1. Doesn't really help us, but I like the idea of `queso-ui` staying pretty up to date. As we add more features, it'll be nice to reach for them and have them there without having to update. The build tasks got a security patch which didn't do much, but I figured update while I'm in there.

2. I hated the names I created because I couldn't remember them and they contradicted the naming convention on the js tasks. The prefix of `css:` is little misleading since it does icons too, but eh feels close enough.

#### How should this be manually tested?

`make backing`
`make`
`make restart`

Just make sure it all builds and the styles aren't broken on /donate

#### How should this change be communicated to end users?

Shouldn't

#### Are there any smells or added technical debt to note?
nope

#### What are the relevant tickets?
off-sprint; general housekeeping


#### Have you done the following, if applicable:
***(optional: add explanation between parentheses)***


#### TODOs / next steps:
